docker-compose.debug.yaml

```diff
@@ -27,3 +27,13 @@ services:
     networks:
       - airbyte_internal
       - airbyte_public
+  worker:
+    environment:
+      - DEBUG_CONTAINER_IMAGE=${DEBUG_CONTAINER_IMAGE}
+      - DEBUG_CONTAINER_JAVA_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005
+  server:
+    # You will need to create a remote JVM debugging Run Configuration
```

Comment:
I personally use the Ubuntu OS and never had issues with remote debugging (just added this var to the required dockerfile), but for MAC users there may be some issues with IPs. I've heard that this tool may help https://github.com/chipmk/docker-mac-net-connect
Maybe it would be worth also this link to notes. @suhomud knows better how it works on MAC.

Reply:
@etsybaev good call out, definitely need that tool for the mac setup

The intent of this change is that you would no longer need to make any changes to `Dockerfile`s since those changes should never be committed.